app/controllers/settings_controller.rb

```diff
@@ -0,0 +1,10 @@
+class SettingsController < ApplicationController
```

Comment:
Maybe we can simplify the code and remove this controller, alongside its routes and abilities :thinking: (see the comment on the view file).